src/epd_driver/include/epd_driver.h

```diff
@@ -177,6 +185,18 @@ typedef struct {
 /** Initialize the ePaper display */
 void epd_init(enum EpdInitOptions options);
 
+/** Rotation getter */
+uint8_t epd_get_rotation();
+
+/** Rotation setter */
```

Comment:
Same here, rotation getter and setter doesn't explain very much.

Reply:
Please check if [this is better](https://github.com/martinberlin/epdiy-rotation/commit/1bbc2d477bf8d7447968f87218b394d401945459)